wordpress-mobile/WordPress-iOS
# Fix/10230 draftextension name

Fixes #10230 

@bummytime Can I bother you with a review? 

Localise the name of the Share Extension. I am not completely sure about adding the string to the `InfoPListTranslator`, and if that would be enough for the localisations to be picked up and uploaded to glotpress. @jtreanor your name was suggested by Github as well, if you don't mind taking a look and if you have any insight over this, that would be great

Before and after
![extension](https://user-images.githubusercontent.com/2722505/50875797-ee2f1b00-1404-11e9-9f16-5df5e41a301c.jpg)


To test:
- Checkout the branch
- Set the simulator language to Spanish. It has to be Spanish, as it is the only language that I have updated the localisation for.
- Install the app, or run the `WordPressShareExtension` scheme.
- Try to share some content (i.e. a web page). Notice the name of the extension localised in Spanish.

Update release notes:
